ngraph/test/files/paddlepaddle/gen_scripts/generate_yolo_box.py

```diff
@@ -0,0 +1,202 @@
+#
```

Comment:
Model generator script is intended to just generate the model for C++ unit test, and to not perform any complex verification.

Reply:
@ceciliapeng2011, could we only contain the model generator codes?